### PR TITLE
Update for WPILib 2023

### PR DESCRIPTION
**Pull request by stephenjust**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -162,27 +152,27 @@ tasks.getByName('eclipseFactorypath') {
...
-
-        whenMerged { classpath ->
-            classpath.entries.each { entry -> 
-                if (entry.path.contains('bin/generated')) {
-                    entry.entryAttributes['ignore_optional_problems'] = true
-                }
-            }
-        }
-    }
+	//synchronizationTasks eclipseJdtApt, eclipseJdt, eclipseFactorypath
```
Comment (aschokking) [inline build.gradle:155]: ⭐ just delete these?

File: src/test/java/xbot/edubot/rotation/GoToOrientationTest.java [hunk 2]
```diff
@@ -11,7 +11,7 @@ public class GoToOrientationTest extends BaseOrientationEngineTest implements Se
 
     @Test
     public void testGoToOrientation0to150() {
-        DriveToOrientationCommand command = new DriveToOrientationCommand(this.drive);
+        DriveToOrientationCommand command = this.getInjectorComponent().driveToOrientationCommand();
```
Comment (aschokking) [inline src/test/java/xbot/edubot/rotation/GoToOrientationTest.java:14]: ❔ out of curiosity, why move away from building the commands manually here?

**Review by aschokking — APPROVED** [review]

File: src/test/java/xbot/edubot/rotation/GoToOrientationTest.java (hunk 2 above)
Comment (stephenjust) [reply src/test/java/xbot/edubot/rotation/GoToOrientationTest.java:14]: Students were getting super confused in the last exeecises when changing constructors.

**Comment by aschokking** [comment]
@stephenjust any reason not to merge this?

**Comment by stephenjust** [comment]
Not that I'm aware

**Review by JohnGilb — APPROVED** [review]

**Pull request merged**